src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -902,6 +901,11 @@ public Locomotive waitForCondition(ExpectedCondition<?> condition, long timeOutI
         return this;
     }
 
+
+    public Locomotive waitUntilNotPresent(String id) {
+        return waitForCondition(ExpectedConditions.invisibilityOfElementLocated(PageUtil.buildBy(configuration, id)));
```

Comment:
What does the error messaging look like for this?  Consider adding some plain text error handling if it's not immediately recognizable from this method since there is some waiting involved.